test/tests/benchmark/runtime_tests.py

```diff
@@ -0,0 +1,118 @@
+from config.benchmark_config import *
+from config.amqp import *
+from modules.logger import Log
+from modules.amqp import AMQPWorker
+from on_http import NodesApi as Nodes
+from on_http import WorkflowsApi as Workflows
+from on_http import rest
+from datetime import datetime
+from proboscis.asserts import assert_equal
+from proboscis.asserts import assert_false
+from proboscis.asserts import assert_raises
+from proboscis.asserts import assert_not_equal
+from proboscis.asserts import assert_is_not_none
+from proboscis.asserts import assert_true
+from proboscis import SkipTest
+from proboscis import test
+from json import loads
+
+from ansible_control import ansibleControl
+import json
+import time
+
+from data_display import DataDisplay
+
+LOG = Log(__name__)
+
+@test(groups=['runtime.tests'])
+class RuntimeTests(object):
+
+    def __init__(self):
+        self.__client = config.api_client
+        self.__worker = None
+        self.__discovery_duration = None
+        self.__discovered = 0
+        self.__testname = 'runtime'
+        self.__ansible = ansibleControl()
+
+    def start_daemon(self):
+        return self.__ansible.run_playbook('start_daemon.yml')
+
+    def collect_data(self):
+        self.__ansible.render_log_name(self.__testname)
+        return self.__ansible.run_playbook('collect_data.yml')
+
+    def check_compute_count(self):
+        Nodes().api1_1_nodes_get()
+        nodes = loads(self.__client.last_response.data)
+        count = 0
+        for n in nodes:
+            type = n.get('type')
+            if type == 'compute':
+                count += 1
+        return count
+
+    @test(groups=['runtime.tests', 'check_precondition'])
```

Comment:
Instead of reimplementing test cases here, add a `depends_on_groups=['api-v1.1', 'api-v2.0', 'api-redfish-1.0']` or set a subset of test groups that you want to benchmark.